build-requirements.txt

```diff
@@ -0,0 +1,5 @@
+bumpversion~=0.6.0
+setuptools~=70.0
+twine~=5.0
```

Comment:
We were installing this directly and upgrading, so we never actually used 4.0. 4.0 is also broken, see https://github.com/pypa/twine/issues/977. From the post in there, the date of the fix, and the dates of the releases, the fix was released as part of 5.0.